server/interfaces/prisonApi/csraAssessment.ts

```diff
@@ -11,12 +14,15 @@ export interface CsraAssessment {
   assessmentSeq: number
   assessorUser?: string
   bookingId: number
+  calculatedClassificationCode?: ClassificationCode
   cellSharingAlertFlag: boolean
-  classificationCode?: 'LOW' | 'MED' | 'STANDARD' | 'HI'
+  classificationCode?: ClassificationCode
   classificationReviewReason?: string
   nextReviewDate?: string
   offenderNo: string
-  originalClassificationCode?: 'LOW' | 'MED' | 'STANDARD' | 'HI'
+  originalClassificationCode?: ClassificationCode
+  overrideReason?: string
+  overridingClassificationCode?: ClassificationCode
   questions: [
     {
       question: string
```

Comment:
Does this marry up with the prison api models? Seems like a lot of optional values

Reply:
yeah it matches up - it's a 2-3 stage process so a lot of things can be undefined at the start